sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonReportSensor.java

```diff
@@ -30,9 +32,6 @@
 import org.sonar.api.resources.Project;
 import org.sonar.api.utils.WildcardPattern;
 
-import java.io.File;
-import java.util.List;
-
 public abstract class PythonReportSensor implements Sensor {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Sensor"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)